src/subset_handler.h

```diff
@@ -0,0 +1,16 @@
+#define MAX_FILE_NAME_SIZE 512
```

Comment:
Move this file to a separate folder for headers.
Maybe headers folder should be inside src folder only. Check some existing open-source repos.